### ISSUE TITLE
Latest release in debian repository stuck at 1.0.5

### ISSUE THREAD

**Issue by kimholan** [getgauge/gauge#1508]
### Expected behavior
apt-get update gauge should update it to the most recent available version, which is 1.0.7 assuming it's supposed to be an official release.

### Actual behavior
it doesn't update past 1.0.5

### Steps to reproduce

```
$ date
Fri 18 Oct 2019 02:56:18 PM CEST
$ sudo apt-get update
Hit:1 http://ppa.launchpad.net/numix/ppa/ubuntu disco InRelease
Hit:2 http://ftp.nl.debian.org/debian buster InRelease                                                                                                                          
Ign:3 http://dl.google.com/linux/chrome/deb stable InRelease                                                                                                                    
Hit:4 http://security.debian.org/debian-security buster/updates InRelease                                                      
Hit:5 http://ftp.nl.debian.org/debian buster-updates InRelease                                                                 
Hit:6 http://dl.google.com/linux/chrome/deb stable Release                                                                     
Ign:7 https://dl.bintray.com/gauge/gauge-deb stable InRelease                                            
Get:8 https://dl.bintray.com/gauge/gauge-deb stable Release [1,840 B]                             
Hit:9 https://download.docker.com/linux/debian buster InRelease
Fetched 1,840 B in 1s (2,706 B/s)
Reading package lists... Done
$ sudo apt-get upgrade gauge
Reading package lists... Done
Building dependency tree       
Reading state information... Done
gauge is already the newest version (1.0.5).
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
$ apt-cache madison gauge
     gauge |      1.0.5 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      1.0.4 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      1.0.1 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      1.0.0 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.8 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.7 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.5 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.4 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.3 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.2 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.9.0 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.8.5 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.8.4 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.8.3 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.8.2 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.8.1 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.8.0 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.7.0 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.6.2 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.6.1 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.6.0 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
     gauge |      0.5.0 | https://dl.bintray.com/gauge/gauge-deb stable/main amd64 Packages
```

Downloading it manually and installing with dpkg works:

```
$ sudo dpkg -i ~/Downloads/gauge-1.0.7-amd64.deb 
(Reading database ... 221008 files and directories currently installed.)
Preparing to unpack .../gauge-1.0.7-amd64.deb ...
Unpacking gauge (1.0.7) over (1.0.5) ...
Setting up gauge (1.0.7) ...
```

### Gauge version

$ gauge --version
Gauge version: 1.0.5
Commit Hash: 562f036

Plugins
-------
html-report (4.0.8)
java (0.7.2)
screenshot (0.0.1)
xml-report (0.2.1)

**Comment by nehashri** [comment]
@kimholan The latest released version of Gauge is 1.0.6. Can you please try updating it again and see if the package is available?

**Comment by kimholan** [comment]
@nehashri : thanks a bunch. it updated from the debian repository now:
```
$ sudo apt-get upgrade gauge
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  gauge
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 4,427 kB of archives.
After this operation, 69.6 kB of additional disk space will be used.
Do you want to continue? [Y/n] Y
Get:1 https://dl.bintray.com/gauge/gauge-deb stable/main amd64 gauge amd64 1.0.6 [4,427 kB]
Fetched 4,427 kB in 0s (11.0 MB/s)
Reading changelogs... Done
(Reading database ... 205031 files and directories currently installed.)
Preparing to unpack .../archives/gauge_1.0.6_amd64.deb ...
Unpacking gauge (1.0.6) over (1.0.5) ...
Setting up gauge (1.0.6) ...
```
